test/workspace/tsconfig.json

```diff
@@ -9,12 +9,12 @@
     "skipLibCheck": true,
     "strictNullChecks": true,
     "noUncheckedIndexedAccess": true,
+    "plugins": [
+      {
+        "name": "@prettify-ts/typescript-plugin"
+      }
+    ]
```

Comment:
This is the only addition needed to have prettified hover additions within your editor. VSCode is currently excluded (it uses the old logic, just to be safe).